### PR TITLE
Remove skip based on Python 3 > 3.4

### PR DESCRIPTION
**Pull request by nickpowersys**

### CODE REVIEW

**Comment by jakirkham** [comment]
The right way to do this to [re-render]( https://github.com/conda-forge/conda-smithy/blob/master/README.md#re-rendering-an-existing-feedstock ). Could you please update this PR accordingly?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py>34]
+    skip: False
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also we still want this `skip`.

**Comment by jakirkham** [comment]
So this part of the Python standard library for newer versions of Python. What is the reason for not wanting to use this from the standard library?

**Comment by nickpowersys** [comment]
I have a repo that is cross-compatible between Python 2 and 3, and have
been using the backport for both 2 and 3 since I started developing under
3.4. I do not want to start further revisions and testing right now for
this.

**Comment by jakirkham** [comment]
Where is the library?

**Comment by nickpowersys** [comment]
https://github.com/nickpowersys/CaaR/

**Comment by nickpowersys** [comment]
Can you please let me know your thoughts? If someone wants to install my package using conda under 3.5, the installation would be prevented. Thanks!

**Comment by jakirkham** [comment]
The typical strategy for this sort of case is to check the `python` version in `setup.py` and only require `configparser` when the Python major version is less than `3`. Here's an [example]( https://github.com/mitsuhiko/flask-openid/blob/b191851b74b0786def15802333d2d94cecefa968/setup.py#L30-L34 ).

**Comment by halldc** [comment]
@jakirkham @ocefpaf @bgruening 

The Apache Airflow project v1.10 also suffers from this problem (see [this line](https://github.com/apache/incubator-airflow/blob/1.10.0/setup.py#L273) and [this line](https://github.com/apache/incubator-airflow/blob/1.10.0/airflow/configuration.py#L38)).

Conda can't install `airflow 1.10` for py35 and py36 until `configparser` is available.

**Comment by halldc** [comment]
Hi @ocefpaf - thanks for taking a look at this!

I can see you've made it a `noarch` package, which makes sense. But you still require `python <3.6`, which blocks conda from installing `airflow v1.10` with py36.

**Comment by ocefpaf** [comment]
> I can see you've made it a `noarch` package, which makes sense. But you still require `python <3.6`, which blocks conda from installing `airflow v1.10` with py36.

My updating script uses the package metadata and it was fed:

```
    classifiers=[
        'Development Status :: 5 - Production/Stable',
        'Intended Audience :: Developers',
        'License :: OSI Approved :: MIT License',
        'Natural Language :: English',
        'Operating System :: OS Independent',
        'Programming Language :: Python',
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.6',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
        'Topic :: Software Development :: Libraries',
        'Topic :: Software Development :: Libraries :: Python Modules',
    ],
```

If you are 100% sure it works with `Python >3.5` we can make that change, but I don't have time to test/check it now.

**Comment by halldc** [comment]
I've been using Airflow 1.10 with Python 3.6.6 for a couple of months now without any issues (installed using pip instead of conda). This means I've been using `backports.configparser` with Python 3.6.6 during this time.

**Pull request merged**